server/roomlogs.js

```diff
@@ -12,6 +12,9 @@
 /** @type {typeof import('../lib/fs').FS} */
 const FS = require(/** @type {any} */('../.lib-dist/fs')).FS;
 
+const SPECTATOR_CHANNEL = 5;
+const OMNISCIENT_CHANNEL = 5;
```

Comment:
ummm these should probably not both be `5`.

Also under my proposed change, spectator would be `0` and omniscient would be `-1`.